src/test/java/org/folio/rest/utils/TenantApiTestUtil.java

```diff
@@ -22,11 +22,11 @@ private TenantApiTestUtil() {
 
   }
 
-  public static JsonObject prepareTenantBody(boolean isLoadSampleData, boolean isUpgrade) {
+  public static JsonObject prepareTenantBody(boolean isLoadSampleData, boolean isLoadReferenceData, boolean isUpgrade) {
```

Comment:
I don't think we need the isUpgrade code anymore, it is legacy code, RMB is capable of this already